tests/integration/conftest.py

```diff
@@ -247,6 +248,19 @@ def unique_dataset(client, rand_gen):
     dataset.delete()
 
 
+@pytest.fixture
```

Comment:
we also have a  small_dataset fixture in tests/integration/test_batch.py 
If you have time, can you make sure both this test and that use the same fixture?

Reply:
Good catch. Will do that.